### PR TITLE
Add specific message for trying to run MGCB or 2MGFX from 32 bit OS

### PR DESCRIPTION
**Pull request by cra0zy**

### CODE REVIEW

**Comment by KonajuGames** [comment]
I'm happy with that.

File: Tools/2MGFX/Program.cs [hunk 1]
```diff
@@ -7,6 +7,12 @@ public static class Program
     {
         public static int Main(string[] args)
         {
+            if (!Environment.Is64BitOperatingSystem)
```
Comment (tomspilman) [inline Tools/2MGFX/Program.cs]: Would this be safer as `Environment.Is64BitProcess`? 

https://msdn.microsoft.com/en-us/library/system.environment.is64bitprocess(v=vs.110).aspx

Comment (cra0zy) [inline Tools/2MGFX/Program.cs]: How would that be safer?

Comment (tomspilman) [inline Tools/2MGFX/Program.cs]: There are several ways to force a AnyCPU assembly to run as 32bit on a 64bit OS.  This error message would be skipped in those cases.  What we actually want to check is if the process is 64bit or not.

**Comment by tomspilman** [comment]
Thanks @cra0zy !  Merging,

**Pull request merged**